meal-planner/src/App.js

```diff
@@ -1,19 +1,78 @@
-import styled from 'styled-components'
-import style from './App.css'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import woodBackground from './assets/wood_background.jpg'
+import logo from './assets/logo.png'
+import Button from './components/Button'
+import MealSelect from './components/MealSelect'
 
 export default function App() {
+    const weekdays = [
+        'Montag',
+        'Dienstag',
+        'Mittwoch',
+        'Donnerstag',
+        'Freitag',
+        'Samstag',
+        'Sonntag',
+    ]
+
+    const [isWeekday, setIsWeekday] = useState()
```

Comment:
Seen like your're going to fill the hook with a weekday, `isWeekday` sounds like a boolean (not day of weekend??? true/false). What about something like `selectedWeekday`?

Reply:
good point